docs/ocis/migration.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "Migration"
+date: 2021-03-16T16:17:00+01:00
+weight: 41
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis
+geekdocFilePath: migration.md
+---
+
+
+## Migration
+
+The migration happens in subsequent stages while the service is online.
+
+### User Stories
+As an admin I need to avoid downtime.
+As an admin I want to migrate certain groups of users before others.
+As a user I need a seamless migration and not loose data by any chance.
+
+### Stage-0
+Is the pre-migration stage having a functional ownCloud 10 instance
+
+### Stage-1
+Introduce OpenID Connect to server and Clients
+
+### Stage-2
+Introduce new web ui, let end users get used to it voluntarily
+
+### Stage-3
+start oCIS backend and make read only tests on existing data using owncloud storage driver and database backed share manager
+
+### Stage-4
+test updating data using oCIS backend
+
+### Stage-5
+introduce reverse proxy and switch over early adoptors, let admins gain trust in the new backend
```

Comment:
"let admins gain trust" sounds blurry. What do we mean by that? Are there any way of verifying consistency of the backend for example?